source/content/guides/drush/02-drush-versions.md

```diff
@@ -109,6 +131,4 @@ Configure a newer version of Drush as [documented above](#configure-drush-versio
 ## More Resources
 
 - [Avoiding “Dependency Hell” with Site-Local Drush (Blog)](https://pantheon.io/blog/avoiding-dependency-hell-site-local-drush)
-- [Fix Up Drush Site Aliases with a Policy File (Blog)](https://pantheon.io/blog/fix-drush-site-aliases-policy-file)
```

Comment:
@greg-1-anderson these blog posts didn't get manually copied from our old D7 site to D10. We could still copy them if you think they're relevant still